poster/poster.tex

```diff
@@ -361,7 +367,7 @@ \section*{Conclusion}
 
 {\tt stginga} utilizes Ginga plugins to support HST and JWST data analysis,
 which includes background subtraction, bad pixel correction, DQ flags
-inspection, and signal-to-noise calculations.
+inspection and signal-to-noise calculations.
```

Comment:
I would like to keep the Oxford comma here.

Reply:
:smile_cat: